kong/plugins/prometheus/exporter.lua

```diff
@@ -48,6 +48,9 @@ local function init()
       "Number of Stream connections",
       {"state"})
   end
+  metrics.timers = prometheus:gauge("nginx_current_timers",
```

Comment:
```suggestion
  metrics.timers = prometheus:gauge("nginx_timers",
```
I _think_ there is no such thing as "current", it is either running or pending. Best avoided else we risk further confusion.

Reply:
Good idea, done.